### ISSUE TITLE
The stack trace on android does not show readable stack trace, while ios does

### ISSUE THREAD

**Issue by jarvisluong** [getsentry/sentry-react-native#1172]
**OS:**
- [ ]  Windows
- [x]  MacOS
- [ ]  Linux

**Platform:**
- [ ]  iOS
- [x]  Android

**SDK:**
- [x]  `@sentry/react-native` (>= 1.0.0)
- [ ]  `react-native-sentry` (<= 0.43.2)

**SDK version:** @sentry/react-native@^1.7.2

**`react-native` version:** 0.62.2

**Are you using Expo?**
- [ ]  Yes
- [x]  No

**Are you using sentry.io or on-premise?**
- [x]  sentry.io (SaaS)
- [ ]  on-premise

**If you are using sentry.io, please post a link to your issue so we can take a look:**

[Link to issue](https://sentry.io/share/issue/c022322287da449dadd2f0ba95f9b9f7/)

**Configuration:**

(`@sentry/react-native`)
```
Sentry.init({
    dsn:
      "https://...@....ingest.sentry.io/..."
  });
```

---
**I have following issue:**

Crash log on android does not produce readable stack trace like iOS (source map is even detected)

**Steps to reproduce:**
- Step 1: Create a new react native project an integrate @sentry/react-native
- Step 2: Write a JS crash
- Step 3: Build an android apk, and ios app
- Step 4: Run the APK and ios app and cause the app to crash

**Actual result:**

The stack trace on android does not show readable stack trace, while ios does

https://sentry.io/share/issue/c022322287da449dadd2f0ba95f9b9f7/

**Expected result:**

The stack trace should be readable for both android and ios

**Comment by jarvisluong** [comment]
It seems that the link show the stack trace for iOS, I attach the screenshot of the same issue but for android:

https://imgur.com/a/ohGU9Qe

**Comment by jennmueng** [comment]
@jarvisluong Hmm can I see the event's raw JSON? This just might be all that is being sent from React Native to our SDK, because usually if stack traces aren't detected or not working it would show raw minified code in the stack trace.

What you can try is inside `Sentry.init`, add this:
```js
Sentry.init({
  ...
  beforeSend: (event) => ({
    // log event or paste it into a text box
  })
  ...
});
```
and see inside `exception.value` whether it contains detailed stack trace frames or not.

**Comment by jarvisluong** [comment]
You are correct 🤔 @jennmueng , it seems that the stacktrace in the device itself is already non-readable. Is there anything I could do to fix this issue?

https://imgur.com/a/qx8Txk9

**Comment by jarvisluong** [comment]
Looks like this is the same issue: https://github.com/getsentry/sentry-react-native/issues/755

**Comment by cameron-quad** [comment]
@jarvisluong was having this issue but then I read the [android specifics ](https://docs.sentry.io/platforms/react-native/#android-specifics) , I had `org.gradle.configureondemand=true`,  so removing or setting to false got it working again. Check that

**Comment by jarvisluong** [comment]
> @jarvisluong was having this issue but then I read the [android specifics ](https://docs.sentry.io/platforms/react-native/#android-specifics) , I had `org.gradle.configureondemand=true`, so removing or setting to false got it working again. Check that

Thanks! I don't have that settings in my gradle.properties, and our builds are clean everytime in the CI, so it shouldn't be the case

**Comment by marandaneto** [comment]
does that still happen using the latest Sentry RN SDK? I can't reproduce it

**Comment by shashigit107** [comment]
@jarvisluong  same issue I am facing how did you solved this issue

**Comment by shashigit107** [comment]
> [@jarvisluong](https://github.com/jarvisluong) was having this issue but then I read the [android specifics ](https://docs.sentry.io/platforms/react-native/#android-specifics) , I had `org.gradle.configureondemand=true`, so removing or setting to false got it working again. Check that

in gradles.properties . dont have this code : org.gradle.configureondemand=true
